src/MessageBox.Avalonia/ViewModels/Commands/RelayCommand.cs

```diff
@@ -7,7 +7,7 @@ public class RelayCommand : ICommand
     {
         private readonly Func<object, bool> canExecute;
         private readonly Action<object> execute;
-        private EventHandler? _canExecuteChanged;
```

Comment:
Indeed, but from my understanding EventHandler is already a nullable type, so there is no reason to add `?` again here. I've also never saw implementations doing it like this. Do you see a problem here I've may overseen?

Reply:
@Fruchtzwerg94, to me, `EventHandler` doesn't look any special. E.g. this program will compile with a warning on .NET 6 SDK (`<Nullable>enable</Nullable>`):
```csharp
class Foo
{
    private EventHandler _foo; // Warning CS8618 : Non-nullable field '_foo' must contain a non-null value when exiting constructor. Consider declaring the field as nullable.
    public Foo()
    {
        Console.WriteLine("Foo::ctor");
    }
}
```

In this particular case, it may not trigger a warning in `RelayCommand` for whatever reason (though, to be honest, I see no reason to not trigger the warning).

And even more: see this warning.
```csharp
class Foo
{
    private EventHandler _foo;
    
    public event EventHandler Fooh
    {
        add => _foo += value;
        remove => _foo -= value; // Warning CS8601 : Possible null reference assignment.
    }
    
    public Foo()
    {
        Console.WriteLine("Foo::ctor");
    }
}
```

So, in my case, the compiler properly reacts on `-=` operator possibly producing a `null` reference for a delegate-typed variable.

To summarize: yes, I still believe this particular change is incorrect from the nullability point of view, and it should trigger a couple of compiler warnings.